Scripts/Scripts.csproj

```diff
@@ -27,4 +27,8 @@
   <ItemGroup>
     <Reference Include="System.Web" />
   </ItemGroup>
-</Project>
\ No newline at end of file
+
+  <ItemGroup>
+    <PackageReference Include="IPNetwork2" Version="2.5.329" />
```

Comment:
I don't see this used?

Reply:
The nuget package is called IPNetwork2:
https://github.com/lduchosal/ipnetwork

Reply:
I meant I don't see the nuget package used in the PR, unless I missed something?

Reply:
The package is called IPNetwork2 but the class is called IPNetwork within the System.Net namespace:
https://github.com/lduchosal/ipnetwork/blob/master/src/System.Net.IPNetwork/IPNetwork.cs